source/_integrations/sensor.rest.markdown

```diff
@@ -222,7 +222,7 @@ sensor:
 
 {% endraw %}
 
-### Accessing an HTTP authentication protected endpoint
+### Accessing a HTTP authentication protected endpoint
```

Comment:
There are quite a few debates in general on this one. From that perspective, I would say neither are wrong.

That said, looking at the bigger sources in this spectrum, Wikipedia, CloudFlare, Mozilla ; they use `an HTTP`.